types/ensure/file.pp

```diff
@@ -0,0 +1,39 @@
+# ensure
```

Comment:
IMHO this should live in stdlib.

Reply:
I've added the ensure types to https://github.com/puppetlabs/puppetlabs-stdlib/pull/1006 , I'll remove them from this PR